Tasks/Common/azure-blobstorage-artifactProvider/azureBlobStorageProvider.ts

```diff
@@ -61,7 +66,12 @@ export class AzureBlobProvider implements models.IArtifactProvider {
 
     public getArtifactItem(artifactItem: models.ArtifactItem): Promise<NodeJS.ReadableStream> {
```

Comment:
Shouldn't getArtifactItems also be changed to add the prefix

Reply:
getArtifactItems removes the prefix, and getArtifactItem appends it